packages/@apphosting/create/src/bin/create.ts

```diff
@@ -1,22 +1,88 @@
 #! /usr/bin/env node
-import { spawn } from "child_process";
 import { program } from "commander";
+import { downloadTemplate } from "giget";
+import { select } from "@inquirer/prompts";
+import spawn from "@npmcli/promise-spawn";
+import ora from "ora";
+
+console.log(process);
```

Comment:
This is temp while I test to see if I can differentiate between `npm create ...`, `yarn create ...`, `npx ...` etc. to see if I can infer the appropriate package manager